src/explore-education-statistics-admin/package.json

```diff
@@ -3,10 +3,10 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
```

Comment:
I've added a new dependency on `@lukemorales/query-key-factory` so we'll need to update accordingly